prospector/datamodel/commit_features.py

```diff
@@ -8,3 +8,4 @@ class CommitFeatures(BaseModel):
     references_vuln_id: bool = False
     time_between_commit_and_advisory_record: int = 0
     changes_relevant_path: bool = False
+    avg_hunk_size: int = 0
```

Comment:
I think that maybe, for the features that are derived from attributes of the commit member object, we could have an initialization in the __post_init__(self) fucntion...

Reply:
Good idea and thanks for the pointers.

Reply:
We move these field initializations into the __post_init__ method in #143 together with other structural improvements, after the pending PRs are merged.